.github/pull_request_template.md

```diff
@@ -2,73 +2,39 @@
 Thank you for contributing to TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
 -->
 
-## What have you changed? (mandatory)
+### What problem does this PR solve? <!--add issue link with summary if exists-->
```

Comment:
how about: "add issue link with summary" and remove "if exists" to force every PR related to an issue.